app/controllers/onboarding_controller.rb

```diff
@@ -25,8 +25,10 @@ def show
   end
 
   def create
+    # FIXME: there is a inherent inconsistency possible here the overall @organization is taken from
+    # the onboarding paths (if scoped) whereas the context of the contributor is taken from the token itself.
```

Comment:
I remember this being discussed, but I'm not sure if it was over the phone, or written.

Do we need to keep the `organization_from_token` at all? I would expect that we would no longer maintain this way of assigning the organization to the contributor.

Would the changes to the routing proposed in these several related PRs invalidate all existing onboarding links? They would not contain the org id, and we don't have a redirect so my assumption would be that it would invalidate all existing onboarding links.

If this is the case, and we don't want to look into how to solve this, I would remove the org id from the jwt creation and remove this.

If we somehow want to maintain this method to extract the organization from the jwt, I would expect that we would have a check that it is the same organization as the one derived from the route and if not to throw an error.

Two questions:
* Are these changes in the onboarding controller somehow related to the requests routes, or should they be moved?
* Is there a use case for sending one link to a contributor and them onboarding for multiple organizations? I guess for now, we are going to keep it simple and the contributor would need to onboard to the orgs separately, with separate links.

Reply:
obsolete fixed in another branch